implementations/cats/src/test/scala/sttp/client3/impl/cats/package.scala

```diff
@@ -8,6 +8,9 @@ import scala.concurrent.Future
 package object cats {
 
   val convertCatsIOToFuture: ConvertToFuture[IO] = new ConvertToFuture[IO] {
-    override def toFuture[T](value: IO[T]): Future[T] = value.unsafeToFuture()
+    override def toFuture[T](value: IO[T]): Future[T] = {
```

Comment:
also not entirely sure what this is for, but I assume it's part of the existing arch